gettingstarted.tex

```diff
@@ -45,13 +45,13 @@ \subsubsection{CTRL}
 
 Holding \specialkey{CTRL} and pressing \megakey{*} enters the Matrix Mode Debugger (refer to the {\bf MEGA65 Book} for more details).
 
-\subsubsection{RUN STOP}
+\subsubsection{RUN/STOP}
 \index{Keyboard!RUN STOP}
-Normally, pressing \specialkey{RUN\\STOP} stops the execution of a program. Holding \specialkey{SHIFT} while pressing \specialkey{RUN\\STOP} {\bf RUN}s the first program from disk.\index{BASIC 65 Commands!RUN}
+The \specialkey{RUN\\STOP} is the RUN / STOP key and is located at the top left-hand side of the keyboard. Normally, pressing \specialkey{RUN\\STOP} stops the execution of a program. Holding \specialkey{SHIFT} while pressing \specialkey{RUN\\STOP} {\bf RUN}s the first program from disk.\index{BASIC 65 Commands!RUN}
```

Comment:
I don't think this new sentence is needed. `\specialkey{RUN\\STOP}` is the key. "The RUN STOP is the RUN /STOP key" is not something that needs to be said, given the styling of this chapter. `\specialkey{RUN\\STOP} is located at the top left-hand side of the keyboard.` is acceptable, but maybe shouldn't be the first sentence here when the keys are introduced by their purpose.

In general I welcome more significant prose rewrites in this chapter, as it's one that I left untouched to not wash away previous contributions with the 2nd ed. update. Personally, I want to redesign this chapter to put key graphics and descriptions in a more columnar layout to make it easier to scan, but that's only one idea.

Reply:
I'll remove this.